docs/core/tools/dotnet-pack.md

```diff
@@ -87,3 +91,7 @@ Packs the current application and place the resulting packages into the specifie
 `dotnet pack --no-build --output nupkgs`
 
 Packs the current project into the specified folder and skips the build step.
+
+`dotnet pack --version-suffix "ci-1234"`
+
+Packs the current application and updates the resulting packages version with the given suffix. E.g. version `1.0.0-*` will be updated to `1.0.0-ci-1234`.
```

Comment:
As far as I can tell, `dotnet pack` is for packing *libraries*, not *applications*. Most of this document talks about a "project", so maybe change "application" to "project" here and also in the `--output` example above?